### ISSUE TITLE
Adding .project and .classpath to case studies

### ISSUE THREAD

**Issue by jabiercoding** [mde-optimiser/case_studies#19]
When we try to import a case study in Eclipse, no .projects are found in this repository. I needed to create a java project (e.g., cra), copy the files, add the xtend nature, add things to the classpath, until it worked... Why not adding directly the .project and .classpath files to have ready-to-be-used examples?

**Comment by alxbrd** [comment]
The case studies are maven projects, try to import them as existing maven projects. That should generate the required dot files for Eclipse.

**Comment by szschaler** [comment]
@alxbrd is this well documented on the website / in the case-study repository? If not, it might be worth adding a sentence or two about 'Installation' (or similar) to the documentation.

**Comment by jabiercoding** [comment]
You are right, if I try to import them as existing Maven projects they appear. Maybe you can then modify the MDEOptimiser website in the "First MDEO project" section. You have "they can be imported as existing Eclipse projects", so you can just add as "existing Maven projects". This will help people not familiar with Maven builds. However, when I imported cra or the scrum example as Maven projects, they still have a lot of errors which are not so obvious to resolve. Maybe you can be more specific in the documentation as well.

**Comment by alxbrd** [comment]
@szschaler @jabiercoding Thanks for the feedback, I will update the website accordingly and hopefully this will make it clear for other users. Initially the projects were indeed Eclipse projects, but this has been changed since then and the website remained with the old text.

@jabiercoding For the errors, if they are about MDEOptimiser dependencies, there is an Eclipse quickfix implemented that will add the required libraries to the classpath in the form of an Eclipse container. See if this helps. The same applies for XTend. If there are even more dependencies missing, please post a screenshot and I will try to suggest a fix(but this shouldn't be the case).

**Comment by szschaler** [comment]
Thanks, @alxbrd . For the quickfix: is this documented on the case-study pages somewhere? We should explicitly mention this. I guess the reason it happens is because we no longer include `.classpath` files. Is there a way of keeping the maven setup while still maintaining the `.classpath` files?

@jabiercoding from your experience downloading the case studies, where would be the best place to document these points? In other words, where did you look first and where might you have missed the information?

**Comment by szschaler** [comment]
I've just run into this problem again. It seems this hasn't been fixed in the website or the readme files yet. @alxbrd is this something you could look into?

**Comment by szschaler** [comment]
Also, when importing these as Maven projects, I get lots of errors:

![image](https://user-images.githubusercontent.com/7057319/117128046-cfeb4700-ad94-11eb-98e4-4a20e50e4765.png)
